js/spreadsheet.js

```diff
@@ -17,7 +17,6 @@ var spreadsheet = function(options) {
   var arrayColumns = [];
   var columnNameCounter = 1; // Counter to anonymous columns names
   var rendered = 0;
-
```

Comment:
@YaroslavOvdii There should be an empty line after the `var` block.

I'm not sure what's complicated about this change at the moment.

There should be an empty line before and after a `var` block, except at the beginning or end of a clause. This has always been the rule.

An empty line also shouldn't have any spaces and should be completely empty. This has been the issue that I needed addressed, but instead the whole line has been removed. cc @sofiiakvasnevska